tools/EFToolsVsReferences.settings.targets

```diff
@@ -11,8 +11,10 @@
     <Reference Include="Microsoft.VisualStudio.Shell.Immutable.12.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.Interfaces.14.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.14.0" />
+    <Reference Include="Microsoft.VisualStudio.Telemetry" />
```

Comment:
Using the NuGet package looks cleanest. I'll update to that. It also prevents you from using API newer than Dev14.